lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -488,6 +488,20 @@ def get_edgecolor(self):
         return self._maybe_depth_shade_and_sort_colors(super().get_edgecolor())
 
 
+def _get_data_scale(X, Y, Z):
+    """
+    Estimate the scale of the 3D data for use in depthshading
+    """
+    def _m(data):
+        return np.power(max(data) - min(data), 2)
```

Comment:
```suggestion
        return np.ptp(data)**2
```
and maybe this is short enough to inline.

Reply:
Nice, I haven't seen that function before. I'll make it inline.